src/klein/_isession.py

```diff
@@ -13,11 +13,6 @@
 
 from ._typing import ifmethod
 
-if TYPE_CHECKING:  # pragma: no cover
-    from ._requirer import RequestLifecycle
-
-    IRequestLifecycleT = Union[RequestLifecycle, "IRequestLifecycle"]
```

Comment:
Getting rid of `IRequestLifecycleT` for consistency in how we handle internal type hacking currently for `zope.interface`, see `_IRequestLifecycle` def below.